webui/react/src/utils/integrations.ts

```diff
@@ -0,0 +1,10 @@
+import { IntegrationType } from 'types';
+
+export const createPachydermLineageLink = (
+  integrationData: IntegrationType,
```

Comment:
Consider having this take the pachyderm config directly -- as mentioned before, it's unlikely that we're going to expand this feature within the near to mid future and even then, that it would be another lineage provider. Passing the pachyderm config directly forces the checks to happen at the caller instead, which also means that we only need to check the integrations in order to display the lineage, and not the integrations and then the resulting url.